lib/turbotrig/turbovec.cpp

```diff
@@ -172,6 +172,10 @@ void euler_from_quat(quaternion_t q, float *phi, float *theta, float *psi)
 }
 
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wold-style-cast"
+#pragma GCC diagnostic ignored "-Wcast-qual"
```

Comment:
If there's any way we could get rid of these, that'd be awesome... I'll take a look at it